modules/uhttp/doc.go

```diff
@@ -30,7 +30,6 @@
 //     "io"
 //     "net/http"
 //
-//     "go.uber.org/fx"
```

Comment:
Remove this file.. Looks like these are remains from https://github.com/uber-go/fx/pull/159